### ISSUE TITLE
presenter: get_forwarding_meetings

### ISSUE THREAD

**Issue by peb-adr** [OpenSlides/openslides-backend#1392]
when opening a motion as a non-admin user (e.g. a:a in the dev setup)
the presenter is requested to:
[{"presenter":"get_forwarding_meetings","data":{"meeting_id":1}}]
and responds 403:
{"success": false, "message": "You are not allowed to perform presenter get_forwarding_meetings Missing permission: motion.can_manage"}

Hence unprivileged user will always see the error message box at the bottom when opening a motion.

I'm not sure if this is intended behavior on the backend side and if the client should handle a 403 response differently.
I'm assuming the client catches all response codes >= 300 and turns them into the message box.
So depending on what the convention is here either the presenter should maybe respond 200 with an empty list
or this issue should be resolved in the client.

@r-peschke do you have thoughts?

**Comment by r-peschke** [comment]
@peb-adr @emanuelschuetze 
From backend side this is the requested behavior, see https://github.com/OpenSlides/OpenSlides/wiki/get_forwarding_meetings.
**IMO** the client shouldn't call the backend without having permission to do like it does usually, i.e. doesn't show the button or whatever to call such a service, if the user has no permission to do. An empty list without  message in my opinion would be misleading.

**Comment by jsangmeister** [comment]
I agree with @r-peschke, the permission should be checked in the client and only afterwards should the request be made. Transferring it to the client.
